src/redux/reducers/authenticationReducer.js

```diff
@@ -0,0 +1,16 @@
+import { AUTHENTICATION, LOGOUT } from '../types';
+
+const initialState = false;
+
+const authenticationReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case AUTHENTICATION:
+      return true;
```

Comment:
a good habit to store users auth object in such a reducer instead of just boolean value